pkgs/top-level/python-packages.nix

```
@@ -1600,6 +1600,8 @@ in {
 
   dugong = callPackage ../development/python-modules/dugong {};
 
+  easysnmp = callPackage ../development/python-modules/easysnmp {};
```

you should avoid passing all pkgs directly, a good way to accidentally bloat your closure, just pass the packages you care about.
```suggestion
  easysnmp = callPackage ../development/python-modules/easysnmp { 
    openssl = pkgs.openssl; 
    net_snmp = pkgs.net_snmp;
  };
```

---

No, pythonPackages's callPackage will fill stuff not found in pythonPackages from pkgs itself so that isn't necessary — it just needs to accept openssl and net_snmp.

---

if there's ever a package added to python-modules with those names, then it will break.

---

I'm fully aware that it will fall back to the pkgs namespace